gallery/visualtests/nxRadio.js

```diff
@@ -76,5 +76,19 @@ describe('NxRadio', function() {
     it('looks disabled when hovered', hoverTest(disabledCheckedSelector));
   });
 
+  describe('Tooltip for NxRadio', function() {
+    const selector = '#nx-radio-example .gallery-example-live',
+        hoverSelector = '#nx-radio-example .gallery-example-live label:nth-of-type(3)';
+    it('has a tooltip on hover when set to true', hoverTest(selector, hoverSelector, true));
```

Comment:
Let's have this one test for the visuals of it and then use actual assertions on the DOM in the two tests below, expecting the `.nx-tooltip` to either be or not be present.  Using more screenshots to verify that is a bit wasteful.

Reply:
> Let's have this one test for the visuals of it and then use actual assertions on the DOM in the two tests below, expecting the `.nx-tooltip` to either be or not be present. Using more screenshots to verify that is a bit wasteful.

Updated here: 89466db8325d301cbfe8cc61516a92123789f831

Reply:
> Updated here: https://github.com/sonatype/sonatype-react-shared-components/commit/89466db8325d301cbfe8cc61516a92123789f831

That lost the test for the one where the prop is set to false

Reply:
> > Updated here: [89466db](https://github.com/sonatype/sonatype-react-shared-components/commit/89466db8325d301cbfe8cc61516a92123789f831)
> 
> That lost the test for the one where the prop is set to false

Added here: abeaf022ac8711ce20477aca59311b5cee67bd01